packages/block-library/src/avatar/edit.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	InspectorControls,
+	useBlockProps,
+	__experimentalUseBorderProps as useBorderProps,
+} from '@wordpress/block-editor';
+import {
+	PanelBody,
+	RangeControl,
+	ResizableBox,
+	ToggleControl,
+} from '@wordpress/components';
+import { __, isRTL } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useUserAvatar, useCommentAvatar } from './hooks';
+
+const AvatarInspectorControls = ( { setAttributes, avatar, attributes } ) => (
+	<InspectorControls>
+		<PanelBody title={ __( 'Avatar Settings' ) }>
+			<RangeControl
+				label={ __( 'Image size' ) }
+				onChange={ ( newSize ) =>
+					setAttributes( {
+						size: newSize,
+					} )
+				}
+				min={ avatar.minSize }
+				max={ avatar.maxSize }
+				initialPosition={ attributes?.size }
+				value={ attributes?.size }
+			/>
+			<ToggleControl
+				label={ __( 'Link to user profile' ) }
```

Comment:
I think that this text may be incorrect for the comments, where the link is often an external website?

Reply:
I thought about it. But also, I changed the website on the user settings and could not see any website link. 

Do we know from where WordPress fetch that information?

Could we rephrase to 'Link to user website or author archive'?